src/devices/acurite.c

```diff
@@ -472,10 +472,12 @@ static int acurite_txr_callback(bitbuffer_t *bitbuf) {
 		tempf = acurite_getTemp(bb[4], bb[5]);
 		tempc = fahrenheit2celsius(tempf);
 		humidity = acurite_getHumidity(bb[6]);
-
+//remove all printf usage
+#if 0
 		printf("%s Acurite 5n1 sensor 0x%04X Ch %c, Msg %02x, Wind %d kmph / %0.1f mph, %3.1F C %3.1F F %d %% RH\n",
 		       time_str, sensor_id, channel, message_type,
 		       wind_speed, wind_speedmph, tempc, tempf, humidity);
+#endif
```

Comment:
Change this to a if (debug_output >= 1) {} block. Change the printf() to fprintf(stderr, ...).